modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/CheckboxTag.java

```diff
@@ -137,6 +137,16 @@ protected String getHydratedModuleName() {
 		return super.prepareProps(props);
 	}
 
+	@Override
+	protected void renderIdAttribute() {
+		// Not sure what to do here
```

Comment:
Nothing :)

Reply:
So it's supposed to stay as it is, just without the comment inside? Without any parameters and any body of the method?

Reply:
When you override or implement a method in java everything in the definition must be the same, so if the implemented/override method receive a param you need to receive it, because if the params changes we'll be talking about overcharging a method.

In this case, due you can get rid of the param in the `BaseContainerTag.rederIdAttribute` and `BaseContainerTag.rederCssClassAttribute` methods, yes, here they will look as they are but without the comment.